Forex-UI.md

```diff
@@ -0,0 +1,68 @@
+<img src="/paidy.png?raw=true" width=300 style="background-color:white;">
+
+# Paidy Take-Home Coding Exercises
+
+## What to expect?
+We understand that your time is valuable, and in anyone's busy schedule solving these exercises may constitute a fairly substantial chunk of time, so we really appreciate any effort you put in to helping us build a solid team.
+
+## What we are looking for?
+**Keep it simple**. Read the requirements and restrictions carefully and focus on solving the problem.
+
+**Treat it like production code**. That is, develop your software in the same way that you would for any code that is intended to be deployed to production. These may be toy exercises, but we really would like to get an idea of how you build code on a day-to-day basis.
+
+## How to submit?
+You can do this however you see fit - you can email us a tarball, a pointer to download your code from somewhere or just a link to a source control repository. Make sure your submission includes a small **README**, documenting any assumptions, simplifications and/or choices you made, as well as a short description of how to run the code and/or tests. Finally, to help us review your code, please split your commit history in sensible chunks (at least separate the initial provided code from your personal additions).
+
+# An in-browser Forex rates
+
+Build a Single Page Application displaying Forex rates information which you get through an api.
+
+## Requirements
+
+We would like to see an in-browser application that creates an interface over the [One-Frame API](https://hub.docker.com/r/paidyinc/one-frame). You can display the information in whatever way you wish to (graphs, icons, animations, ...), but try to make it both appealing as well as insightful for the end-user.
+
+Amongst others, we at least expect the following to be addressed:
+
+- being able to select which currency pair to display the information for
+- when selecting a new currency pair, the data displayed should transition nicely
+- the end-user should be able to refresh the data fetched. If the data is updated, a similar transition would happen.
+- make sure you handle failure modes. Some examples of the latter are:
+  - the api takes a long time before returning data
+  - the api doesn't respond
+
+### The One-Frame service
+
+#### How to run locally
+
+* Pull the docker image with `docker pull paidyinc/one-frame`
+* Run the service locally on port 8080 with `docker run -p 8080:8080 paidyinc/one-frame
+
+#### Usage
+__API__
+
+`GET /streaming/rates?pair={currency_pair_0}&pair={currency_pair_1}&...pair={currency_pair_n}`
```

Comment:
Not that this API will be a streaming one, you only call it once and it serves you a never ending stream of exchange rates.

Reply:
i have not implemented this end point in the mock yet, i wanted to get early feedback on whether this is a good exercise or not before investing more time into it.